readthedocs/templates/core/project_details.html

```diff
@@ -18,9 +18,9 @@
   <div class="module-header">
     <h3>{% trans "Versions" %}</h3>
   </div>
-
+  
```

Comment:
Remove Extra spaces !

Reply:
Well, it has been like that in the original code. I did not try to change that style.

Reply:
The line is fine, I am referring to the extra whitespace you have added.